### DISCUSSION TITLE
How to enable Windows Defender?

### DISCUSSION THREAD

**Discussion by HummyBummel** [memstechtips/UnattendedWinstall#202] [Q&A] [answered]
I installed Windows 11 with UnattendedWinstall  2.1.0
Now I want to enable Windows Defender but its not possible to activate it. It jumps to off immediately.

![grafik](https://github.com/user-attachments/assets/2feb7c11-08b6-4493-a747-a43d82771011)

**Comment by zaid-lol** [comment, marked as answer]
go to file explorer/this pc/local disk (C):/Users/your username/ and scroll down until you find "user customization.ps1'' right click and run with PowerShell. Then click on "enable defender" and restart your pc
